drivers/i3c/i3c_common.c

```diff
@@ -563,6 +563,7 @@ int i3c_device_basic_info_get(struct i3c_device_desc *target)
 		goto out;
 	}
 
+	ret = 0;
```

Comment:
I think this should be moved into the `else` block above in line 562.

Reply:
yes, or an else block on 564

Reply:
changed